ledger/acctupdates.go

```diff
@@ -1804,15 +2202,76 @@ func (au *accountUpdates) commitRound(offset uint64, dbRound basics.Round, lookb
 	// au.accounts.
 	flushcount := make(map[basics.Address]int)
 	creatableFlushcount := make(map[basics.CreatableIndex]int)
+	storageFlushcount := make(map[addrApp]int)
+	storageKeyFlushcount := make(map[appKeyPtr]int)
 
 	var committedRoundDigest crypto.Digest
 
 	if isCatchpointRound {
 		committedRoundDigest = au.roundDigest[offset+uint64(lookback)-1]
 	}
 
+	// accountUpdates stores account and app deltas separately, but
+	// catchpoints expect unified deltas, keyed off of address.
```

Comment:
Can you comment on whether the catchpoints should also change to reflect the changes here?